src/components/Menu.jsx

```diff
@@ -12,6 +12,8 @@ const propTypes = {
   renderItemLabel: PropTypes.func,
   value: PropTypes.string,
   onSelect: PropTypes.func,
+  shouldTransferFocus: PropTypes.bool,
+  notifyBeforeFocusWrap: PropTypes.func,
 };
```

Comment:
I think I see what you're saying, and that's not that the callback will override the logic of the keydown handler entirely but instead be called once the focusIndex has been incremented or de-incremented.